README.md

```diff
@@ -128,6 +128,7 @@ before you run start the deployment.
 | `OCM_BASE_URL`                          | OCM API URL used to communicate with OCM and AMS, default: https://api.integration.openshift.com/ |
 | `OCM_CLIENT_ID`                         | ID of Service Account used to communicate with OCM and AMS for Agent Auth and Authz |
 | `OCM_CLIENT_SECRET`                     | Password of Service Account used to communicate with OCM and AMS for Agent Auth and Authz |
+| `JWKS_URL`                              | URL for retrieving the JSON Web Key Set (JWKS) used for verifying JWT tokens in authentication. Defaults to https://api.openshift.com/.well-known/jwks.json | 
```

Comment:
this is not the default right ?

Reply:
It is currently, should we change it to be the working param? didnt want to change current configurations as mentioned above

Reply:
yes looks like it doesn't work